tox.ini

```diff
@@ -264,6 +264,7 @@ deps =
     PyYAML==5.4.1
 commands = python {toxinidir}/scripts/check_doc_links.py
            python {toxinidir}/scripts/check_doc_ipfs_hashes.py
+           python {toxinidir}/scripts/check_ipfs_hashes_pushed.py
```

Comment:
same as 236 how is this invocated?

Reply:
https://github.com/valory-xyz/open-aea/pull/236#discussion_r931935833